src/Parser/ParserUtil.php

```diff
@@ -88,6 +88,12 @@ public static function parseArgument(Lexer &$lexer)
         } else {
             self::syntaxError('no valid argument found');
         }
+        if ($string === 'true') {
```

Comment:
It's in ``parseArgument`` since ``getString`` also gets used to grab non-argument strings (ie. field names).